modules/UI/videolayout/VideoLayout.js

```diff
@@ -177,7 +182,17 @@ const VideoLayout = {
         const currentContainerType = largeVideo.getCurrentContainerType();
         const isOnLarge = this.isCurrentlyOnLarge(id);
         const state = APP.store.getState();
-        const videoTrack = getTrackByMediaTypeAndParticipant(state['features/base/tracks'], MEDIA_TYPE.VIDEO, id);
+        const participant = getParticipantById(state, id);
+        const tracks = state['features/base/tracks'];
+
+        let videoTrack;
+
+        if (getSourceNameSignalingFeatureFlag(state) && participant?.isFakeScreenShareParticipant) {
```

Comment:
I see there's exactly the same logic here:
https://github.com/jitsi/jitsi-meet/pull/11058/files#diff-a70e15a6ec7c3e44fba3f853c6428d2941aa013fab10887136762f9b9af28fadR244
Does it make sense to extract it into utility method?